tests/migrate/test_plural.py

```diff
@@ -74,6 +75,55 @@ def test_plural_too_many_variants(self):
         )
 
 
+class TestPluralOrder(MockContext):
+    # Plural categories corresponding to Lithuanian (lt).
+    plural_categories = ('one', 'few', 'other')
```

Comment:
Please change this to `one, other, few` as per https://hg.mozilla.org/l10n/compare-locales/file/tip/compare_locales/plurals.py#l22.